lib/infosections.py

```diff
@@ -0,0 +1,78 @@
+
+from lib.network import get_ip, get_tor_address
+from lib.qr_generator import generate_qr_code
+from lib.lnd import LndGRPC
+from lib.btc import BtcRPC
+import lib.rpc_pb2 as ln
+import lib.rpc_pb2_grpc as lnrpc
+from lib.network import get_tor_address
+from consts import bold_font
+
+import pygame
+
+class InfoSectionsList():
+
+    def __init__(self):
+        # Init RPC connections
+        self.btc_grpc = BtcRPC()
+        self.lnd_grpc = LndGRPC()
+    class torURL():
+        def __init__(self, parent):
+            self.title = "Tor URL"
+            self.data = "Loading..."
+            self.parent = parent
+        def updateData(self):
+            self.data = get_tor_address()
+        def getData(self):
+            self.updateData()
+            return (self.title, self.data, True, pygame.freetype.Font(bold_font, 20))
+    class maxSend():
```

Comment:
It's a style thing, but for me I like to have spaces between all the classes:
```
class z: 
   def thing:
      do stuff
   def otherthing:
      do more stuff
class y: 
...      
```

I prefer this: 
```
class z: 
   def thing:
      do stuff

   def otherthing:
      do more stuff

class y: 
...      
```

Reply:
🎨  (Just a style change)